.github/ISSUE_TEMPLATE/project.md

```diff
@@ -21,35 +21,56 @@ assignees: ''
 
 <!-- Please fill out the following questions to rationalize the prioritization of this project -->
 
-- Does this serve our mission?
+- Does this serve our mission
 
 - Will this help to make us financially sustainable within a year?
 
-- Is it feasible to build and maintain with our team and runway?
-
 - Can this make us move faster?
 
-- Does this increase quality and not technical debt?
+- Does this increase quality? 
+
+- Does this add technical debt?
```

Comment:
Nice question.
I tend to believe that projects are always ideal and technical debt is a result of shortcuts within the project.

Reply:
That's a fair point. Often we take shortcuts during a project and accrue debt in trade for speed. I guess what I meant here is whether this is going to create organisational overhead that needs to be _staffed_. I'll amend.

Reply:
```suggestion
- Does this add additional overhead that needs to staffed (i.e. checking reports, approving checks etc.)
```

Reply:
@kewitz do you think this is better?